prefect_ray/task_runners.py

```diff
@@ -175,7 +181,7 @@ def exchange_prefect_for_ray_future(expr):
             return_data=True,
         )
 
-        return kwargs_ray_futures
+        return kwargs_ray_futures, upstream_ray_obj_refs
 
     @staticmethod
     def _run_prefect_task(func, *args, **kwargs):
```

Comment:
Can we rename `*args` to `*upstream_ray_obj_refs` and add a note about what's going on here?